README.md

```diff
@@ -49,6 +49,34 @@ npx hardhat deploy  --network localhost
 
 Hardhat re-uses old deployments, to force re-deploy add the `--reset` flag above
 
+## ZKLLVM output check
+
+Place folder with ZKLLVM output to `contracts/zkllvm` directory.
+
+Deploy contracts
+```
+npx hardhat deploy
+```
+
+Verify all folders from `contracts/zkllvm` director
+```
+npx hardhat verify
+```
+
+Verify one folder from `contracts/zkllvm` directory
+```
+npx hardhat verify-one folder-name
```

Comment:
Rename to verify_gate <folder_name>

Reply:
I also don't like verify_gate naming.
As in previous comment we verify whole proof with all arguments.
And even if only gate argument is computed, there are a lot of gates in each gate-argument.
Verify_gate sounds like we verify only one of them.

Maybe `verify-one-circuit-proofs`?